azure/terraform/appservice.rego

```diff
@@ -667,4 +667,361 @@ app_service_remote_debugging_disabled_metadata := {
+    "Policy Help URL": "",
+    "Resource Help URL": "https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/app_service"
+}
+
+
+#
+# PR-AZR-0081-TRF
+#
+
+default app_service_php_version_latest = null
+
+azure_attribute_absence["app_service_php_version_latest"] {
+    resource := input.resources[_]
+    lower(resource.type) == "azurerm_app_service"
+    not resource.properties.site_config
+}
+
+azure_attribute_absence["app_service_php_version_latest"] {
+    resource := input.resources[_]
+    lower(resource.type) == "azurerm_app_service"
+    site_config := resource.properties.site_config[_]
+    not site_config.php_version
+}
+
+azure_issue["app_service_php_version_latest"] {
+    resource := input.resources[_]
+    lower(resource.type) == "azurerm_app_service"
+    site_config := resource.properties.site_config[_]
+    to_number(site_config.php_version) != 7.4
+}
+
+# we need to make it pass if property is missing, as azurerm_app_service may not need php
+app_service_php_version_latest {
+    azure_attribute_absence["app_service_php_version_latest"]
+    not azure_issue["app_service_php_version_latest"]
+}
+
+app_service_php_version_latest {
+    lower(input.resources[_].type) == "azurerm_app_service"
+    not azure_attribute_absence["app_service_php_version_latest"]
+    not azure_issue["app_service_php_version_latest"]
+}
+
+app_service_php_version_latest = false {
+    azure_issue["app_service_php_version_latest"]
+}
+
+app_service_php_version_latest_err = "Azure App Service currently dont have latest version of PHP" {
+    azure_issue["app_service_php_version_latest"]
+}
+
+app_service_php_version_latest_metadata := {
+    "Policy Code": "PR-AZR-0081-TRF",
+    "Type": "IaC",
+    "Product": "AZR",
+    "Language": "Terraform",
+    "Policy Title": "Azure App Service PHP version should be latest",
+    "Policy Description": "This policy will identify the Azure app service which dont have latest version of PHP and give alert",
+    "Resource Type": "azurerm_app_service",
+    "Policy Help URL": "",
+    "Resource Help URL": "https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/app_service"
+}
+
+
+#
+# PR-AZR-0083-TRF
+#
+
+default app_service_python_version_latest = null
+
+azure_attribute_absence["app_service_python_version_latest"] {
+    resource := input.resources[_]
+    lower(resource.type) == "azurerm_app_service"
+    not resource.properties.site_config
+}
+
+azure_attribute_absence["app_service_python_version_latest"] {
+    resource := input.resources[_]
+    lower(resource.type) == "azurerm_app_service"
+    site_config := resource.properties.site_config[_]
+    not site_config.python_version
+}
+
+azure_issue["app_service_python_version_latest"] {
+    resource := input.resources[_]
+    lower(resource.type) == "azurerm_app_service"
+    site_config := resource.properties.site_config[_]
+    to_number(site_config.python_version) != 3.4
```

Comment:
I suggest to have the python version as a variable
every time a new version coming out we can update the variable int he rego file to update the test case

Reply:
fixed